### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Chris-Lorch**
added description of bau vmfi script rewrite and simple instructions on how to run.

### CODE REVIEW

File: bau/README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
-# Purpose
+# Outline
 
-This script was created to substitute the MetricRAG_Calculation pipeline as part of the VMFI data drop process.
\ No newline at end of file
+This direcotry contains python scripts created to substitute part of the MetricRAG_Calculation pipeline used in the VMFI data drop process.
```
Comment (J05h-L) [inline bau/README.md:3]: spelling here - directory

File: bau/README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
-# Purpose
+# Outline
 
-This script was created to substitute the MetricRAG_Calculation pipeline as part of the VMFI data drop process.
\ No newline at end of file
+This direcotry contains python scripts created to substitute part of the MetricRAG_Calculation pipeline used in the VMFI data drop process. 
+
+# Code structure
+
+The `main.py` file is used in running the scripts. Small tasks, such as those associated with the stored procedures and the benchmarking calcualtions, are defined in the helper directory as helper scripts. The controller directory contains scripts which execute the helper scripts. The naming convention of the helpers associated with the 6 stored procedures being rewritten have names matching the stored procedures in the VMFI pipeline.
```
Comment (J05h-L) [inline bau/README.md:7]: spelling here - calculations

File: bau/README.md (hunk 1 above)
Comment (Chris-Lorch) [reply bau/README.md]: Fixed

File: bau/README.md (hunk 2 above)
Comment (Chris-Lorch) [reply bau/README.md]: Fixed

**Review by J05h-L — APPROVED** [review]

**Pull request merged**